windows/test-app.js

```diff
@@ -0,0 +1,234 @@
+function copyAndReplace(
+  srcPath,
+  destPath,
+  relativeDestPath,
+  replacements = {}
+) {
+  const fullDestPath = path.join(destPath, relativeDestPath);
+  if (fs.lstatSync(srcPath).isDirectory()) {
+    if (!fs.existsSync(fullDestPath)) {
+      fs.mkdirSync(fullDestPath);
+    }
+    return;
+  }
+
+  const extension = path.extname(srcPath);
+  if (binaryExtensions.indexOf(extension) !== -1) {
+    // Binary file
+    fs.copyFile(srcPath, fullDestPath, (err) => {
+      if (err) {
+        throw err;
+      }
+    });
+  } else {
+    // Text file
+    const srcPermissions = fs.statSync(srcPath).mode;
+    const content = resolveContents(srcPath, replacements);
+    fs.writeFile(
+      fullDestPath,
+      content,
+      {
+        encoding: "utf8",
+        mode: srcPermissions,
+      },
+      (err) => {
+        if (err) {
+          throw err;
+        }
+      }
+    );
+  }
+}
+
+function copyProjectTemplateAndReplace(
+  destPath,
+  nodeModulesPath,
+  reactNativeModulePath,
+  testAppNodeModulePath
+) {
+  if (!destPath) {
+    throw new Error("Need a path to copy to");
+  }
+
+  if (!reactNativeModulePath) {
+    throw new Error("react-native-windows node module is not installed");
+  }
+
+  if (!testAppNodeModulePath) {
+    throw new Error("react-native-test-app node module is not installed");
+  }
+
+  const srcRootPath = path.join(testAppNodeModulePath, windowsDir);
+  const projDir = "ReactTestApp";
+  const projectFilesDestPath = path.join(
+    nodeModulesPath,
+    generatedDir,
+    windowsDir,
+    projDir
+  );
+
+  fs.mkdirSync(projectFilesDestPath, { recursive: true });
+  fs.mkdirSync(path.join(destPath, windowsDir), { recursive: true });
+
+  const manifestFilePath = findClosestPathTo("app.json");
+
+  let bundleDirContent = "";
+  let bundleFileContent = "";
+  //Read path to resources from manifest and copy them
+  const content = fs.readFileSync(manifestFilePath);
+  let resourcesPaths = {};
+  try {
+    resourcesPaths = JSON.parse(content.toString()).resources;
+  } catch (e) {
+    console.log(chalk.red(`Couldn't parse app.json: \n${e.message}`));
+  }
+  resourcesPaths = (resourcesPaths && resourcesPaths.windows) || resourcesPaths;
+  if (Array.isArray(resourcesPaths)) {
+    for (const resource of resourcesPaths) {
+      const resourceSrcPath = path.relative(
+        path.dirname(manifestFilePath),
+        resource
+      );
+      if (fs.existsSync(resourceSrcPath)) {
+        let relativeResourcePath = path.relative(
+          projectFilesDestPath,
+          resourceSrcPath
+        );
+        if (path.extname(resourceSrcPath) === "") {
+          relativeResourcePath = relativeResourcePath.concat("\\**\\*");
+          bundleDirContent = bundleDirContent.concat(
+            relativeResourcePath + ";"
+          );
+        } else {
+          bundleFileContent = bundleFileContent.concat(
+            relativeResourcePath + ";"
+          );
+        }
+      } else {
+        console.log(
```

Comment:
We should use `console.warn()`:
```suggestion
        console.warn(
```